locales/en-US/notes.properties

```diff
@@ -1,12 +1,12 @@
-welcomeTitle=Welcome!
-welcomeText=This is a simple one-page notepad built in to Firefox that helps you get the most out of the Web.
+welcomeTitle2=Hello!
+welcomeText2=Welcome to this one-page notepad built in to Firefox. Browse the web. Jot down notes. It's that simple.
```

Comment:
For strings you should always use proper apostrophes/curly quotes instead of straight single/double quotes. 

That's the rule within Firefox copy, to the point that you would fail tests if you try to land a strings with ' or "", instead of ’ and “”